docs/redoc-vendor-extensions.md

```diff
@@ -59,13 +59,13 @@ Backported from OpenAPI 3.0 [`servers`](https://github.com/OAI/OpenAPI-Specifica
 
 | Field Name     |	Type	       | Description |
 | :------------- | :-----------: | :---------- |
-| x-tagGroups         | [ [Tag Group Object](#tagGroupObject) ] | A list of tag groups |
+| x-tagGroups         | [ [Tag Group Object](#tag-group-object) ] | A list of tag groups |
 
 ###### How to use with Redoc
 `x-tagGroups` is used to group tags in the side menu.
 Before you use `x-tagGroups`, make sure you **add all tags to a group**, since a tag that is not in a group, **is not displayed** at all!
 
-#### <a name="tagGroupObject"></a>Tag Group Object
+#### Tag Group Object
```

Comment:
Removing these headers could break existing bookmarks. I recommend moving them to the line above rather than deleting them. The current heading will produce a fragment of `#tag-group-object` rather than the previous `#tagGroupObject`.

Reply:
```suggestion
<a name="tagGroupObject"></a>
#### Tag Group Object
```

Like this, right?